pkg/operator/controller/ingress_status.go

```diff
@@ -249,33 +140,75 @@ func (c *ingressStatusCache) computeLoadBalancerStatus(ic *operatorv1.IngressCon
 	conditions = append(conditions, operatorv1.OperatorCondition{
 		Type:    operatorv1.LoadBalancerManagedIngressConditionType,
 		Status:  operatorv1.ConditionTrue,
-		Reason:  "HasLoadBalancerEndpointPublishingStrategy",
-		Message: "IngressController has LoadBalancer endpoint publishing strategy",
+		Reason:  "WantedByEndpointPublishingStrategy",
```

Comment:
I threw this in for discussion. Reflecting on the k8s API guidelines I wonder if this condition type should have been "LoadBalancerUnmanaged=True" — too late!

Reply:
>Condition types should indicate state in the “abnormal-true” polarity. For example, if the condition indicates when a policy is invalid, the “is valid” case is probably the norm, so the condition should be called “Invalid”.

Reply:
Any thoughts on the Reason I'm proposing here? I can revert if the old one is better (and am open to new suggestions).

Reply:
No preference.  I'm not sure a reason is strictly necessary for the "normal" state, but what you have is fine.